src/pattern-library/components/patterns/input/InputGroupPage.tsx

```diff
@@ -34,6 +42,32 @@ export default function InputGroupPage() {
                 </InputGroup>
               </div>
             </Library.Demo>
+
+            <Library.Demo
```

Comment:
Rob noted yesterday how nearly-generic a pattern was in `StudentSelector`, an LMS component. Note that this is basically that pattern here: it's very atomic when you get down to it...